src/data/faq.json

```diff
@@ -87,6 +87,117 @@
+                        "anchor": "rat_privileges",
+                        "active": true,
+                        "textblock": [
+                            "A negative test result can serve as proof of the existence of a negative rapid test result for you, if specified by law. However, the recognition of test evidence may vary from state to state. You should therefore inform yourself about the respective criteria in your federal state."
+                        ]
+                    },
+                    {
+                        "title": "How do I act if my rapid test result is positive?",
+                        "anchor": "rat_what_if_positive_result",
+                        "active": true,
+                        "textblock": [
+                            "In case of a positive test result, you should share your test result and immediately take a PCR test to confirm or debunk the result. In case of a positive PCR test, you should share the test result again to alert contacts you may have met between the two tests. This is provided that the contacts also use the Corona Warn App and the risk calculation determines a high risk, based on duration and distance of contact. If the PCR test is negative, you will not have the option to withdraw your positive rapid test and warning."
+                        ]
+                    },
+                    {
+                        "title": "If I see a high-risk encounter in the App, i.e. a red card, should I do a rapid antigen test or PCR-Test?",
+                        "anchor": "rat_red_card_which_test",
+                        "active": true,
+                        "textblock": [
+                            "If the Corona warning app shows you an encounter with increased risk (red tile), you should take a PCR test and quarantine yourself."
+                        ]
+                    },
+                    {
+                        "title": "Are rapid antigen and PCR tests treated differently in the App?",
+                        "anchor": "rat_same_in_cwa",
+                        "active": true,
+                        "textblock": [
+                            "The rapid test is treated equally to the PCR test by the Corona-Warn-App. If you get tested at one of the partners, you will receive your test result via QR code, which you can scan with the Corona-Warn-App. A negative test result will then be displayed in the Corona-Warn-App for 48 hours.",
+                            "In case of a positive test result, you can warn other people within the shortest possible time by sharing your result - as with a PCR test - and uploading it to the Corona-Warn-App server. The positive test result will be displayed in the app until you share it. If you were previously checked in to an event or location via Event Registration, you can share the positive test result along with your check-in."

```

Comment:
```suggestion
                            "In case of a positive test result, you can warn other people within the shortest possible time by sharing your result - as with a PCR test - and uploading it to the Corona-Warn-App server. The positive test result will be displayed in the app until you share it. If you were previously checked in to an event or location via Event Registration, you can share the positive test result along with your check-ins."

```